subsys/bluetooth/controller/ll_sw/ull_conn_internal.h

```diff
@@ -7,17 +7,52 @@
 /* Macro to convert time in us to connection interval units */
 #define RADIO_CONN_EVENTS(x, y) ((u16_t)(((x) + (y) - 1) / (y)))
 
-/* Macro to return PDU time */
+/*
+ * Macros to return correct PDU time
+ * Note: formula is valid for S=8 coding only
+ * see BT spec Version 5.1 Vol 6. Part B, chapters 2.1 and 2.2
+ * for packet formats and thus lengths
+ *
+ * Payload overhead size is the BT address + the MIC
+ */
+#define PAYLOAD_OVERHEAD_SIZE (2+4)
```

Comment:
Add spaces. (2 + 4)

The 2 is not BT address, but Data Channel PDU Header.
Could update description to say it is Data Channel PDU (as opposed to Device Discovery PDU).